Checkout/Checkout/Source/Validation/Validators/CVVValidator.swift

```diff
@@ -12,9 +12,13 @@ public protocol CVVValidating {
     cvv: String,
     cardScheme: Card.Scheme
   ) -> ValidationResult<ValidationError.CVV>
+    
+    func isValid(cvv: String, for scheme: Card.Scheme) -> Bool
```

Comment:
That would depend on the user. The first one is fine if you want to know `Why its not valid`. The new one I added is simply a response to `Is it valid`? We don't take any action based on rejection failure, we just need yes / no. I would argue the first one is excessive anyway as its not actually used properly, but since that user is already on the way out, I stopped trying to refactor into keeping only the cleaner version of the function.